src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,40 @@
+import {
+    Box, Image, Button, Center, ButtonGroup,
+    Grid
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+
+export const NavigationBar = () => {
+    return (
+        <Grid templateColumns='repeat(3, 1fr)' gap={6}>
+            <Box alignSelf="center" justifySelf="start">
+                <Link to="/">
+                    <Image src="/logo.svg" />
+                </Link>
+            </Box>
+            <Center>
+                <ButtonGroup gap="5" variant="link">
+                    <Button>
+                        About Us
```

Comment:
I was wondering if it would be a good idea to use translations: https://react.i18next.com/latest/using-with-hooks - something like that. What do you think?

Reply:
Yeah, we could add that in the future